tools/project_manager/editproject.php

```diff
@@ -304,6 +307,7 @@ public function set_from_post()
         }
 
         $this->original_marc_array_encd = @$_POST['rec'];
+        $this->hold_states = $this->unserialize(@$_POST['hold_states']) ?? [];
```

Comment:
any reason we shouldn't trust the client with whatever hold states it sends here? I can't think of one to be honest.

Reply:
Well, we should never trust user input 😁 

We correctly escape values before putting them into the database, so there is low risk here, but validation is always better and I need to do this when I add this to the API too. Thanks for calling it out.